tests/tee/signed-images/pod-signed-image.yaml.in

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: signed-image-tests
+  annotations:
+    io.katacontainers.config.hypervisor.kernel_params: "agent.aa_kbc_params=cc_kbc::http://kbs-service.trustee-operator-system:8080 agent.image_policy_file=kbs:///default/security-policy/test agent.enable_signature_verification=true"
+spec:
+  runtimeClassName: kata-${TEE}
```

Comment:
Isn't the runtimeClassName `kata-qemu-$TEE` ?

Reply:
These are the runtime classes in virtlab801 (baremetal PoC):
```
oc get runtimeclass
NAME       HANDLER    AGE
kata       kata       10d
kata-snp   kata-snp   10d
```
I'm quite sure we have kata-tdx in TDX machine.
How do you suggest to proceed?

Reply:
I think there is a difference in upstream and downstream w.r.to runtimeClass. Can the complete runtimeClass be parameterised? This will allow customising it for using with CoCo upstream or with OpenShift

Reply:
Will do thanks!